docs/libcudacxx/extended_api/math/round_down.rst

```diff
@@ -0,0 +1,35 @@
+.. _libcudacxx-extended-api-math-round-down:
+
+``round_down`` Round to the previous multiple
+=============================================
+
+.. code:: cuda
+
+   template <typename T, typename = U>
+   [[nodiscard]] __host__ __device__ inline
+   constexpr cuda::std::common_type_t<T, U> round_down(T a, U b) noexcept;
+
+- *Requires*: ``T`` and ``U`` are integral types (including 128-bit integers) or enumerators.
+- *Preconditions*: ``a >= 0`` is true and ``b > 0`` is true.
+- *Returns*: ``a`` rounded to the previous multiple of ``b``. If ``a`` is already a multiple of ``b``, return ``a``.
```

Comment:
```suggestion
- *Returns*: ``a`` rounded down to the largest multiple of ``b`` less than or equal to ``a``. If ``a`` is already a multiple of ``b``, return ``a``.
```

not a hard recommendation, but reading it for the first time with a fresh mind it took me some time to understand it.